ui/job-view/details/tabs/PerformanceTab.jsx

```diff
@@ -16,6 +86,63 @@ export default class PerformanceTab extends React.PureComponent {
         role="region"
         aria-label="Performance"
       >
+        <div className="performance-panel-actions">
+          {
+            // If there is a profiler link, show this first. This is most likely
+            // the primary action of the user here.
+            profilerLink
+          }
+          {
+            // Just to be safe, use the same isPerfTest check the other
+            // "Create Gecko Profile" button uses in the action menu.
+            isPerfTest(selectedJobFull) ? (
+              <button
+                type="button"
+                className={`btn btn-${
+                  // Only make this primary if there is no profiler link.
+                  profilerLink ? 'secondary' : 'primary'
+                } btn-sm`}
+                onClick={this.createGeckoProfile}
+                title={
+                  'Trigger another run of this test with the profiler enabled. The ' +
+                  'profile can then be viewed in the Firefox Profiler.'
+                }
+              >
+                <FontAwesomeIcon icon={faRedo} className="mr-2" />
+                {profilerLink
+                  ? 'Re-trigger performance profile'
+                  : 'Generate performance profile'}
+              </button>
+            ) : null
+          }
+          <a
+            href={getCompareChooserUrl({
+              newProject: repoName,
+              newRevision: revision,
+            })}
+            target="_blank"
+            rel="noopener noreferrer"
+            className="btn btn-secondary btn-sm"
```

Comment:
Please use `btn-darker-secondary` per: https://treeherder.readthedocs.io/accessibility.html#colors

We don't generally use the bootstrap default blue color for buttons. What do you think about using the `text-darker-info` instead?

Reply:
Yikes, that's a shame that bootstrap is not providing accessible colors. I see using the DevTools accessibility tool that this is true. I would like to maintain the relationship of primary to secondary buttons. I made the following change:

Primary button: `btn-darker-secondary`
Secondary buttons: `btn-outline-darker-secondary`